sdk/samples/iot/provisioning/src/paho_iot_provisioning_example.c

```diff
@@ -82,25 +82,27 @@
     (void)printf("\n"); \
   }
 
-// Buffers
-static char x509_cert_pem_file_path_buffer[256];
-static char x509_trust_pem_file_path_buffer[256];
+// Store environment variables
 static char global_provisioning_endpoint_buffer[256];
 static char id_scope_buffer[16];
 static char registration_id_buffer[256];
-static char mqtt_client_id_buffer[128];
-static char mqtt_username_buffer[128];
-static char register_publish_topic_buffer[128];
-static char query_topic_buffer[256];
+static char x509_cert_pem_file_path_buffer[256];
+static char x509_trust_pem_file_path_buffer[256];
 
 // Clients
 static az_iot_provisioning_client provisioning_client;
 static MQTTClient mqtt_client;
+static char mqtt_client_id_buffer[128];
+static char mqtt_username_buffer[128];
+
+// Topics
+static char register_publish_topic_buffer[128];
```

Comment:
Can you please explain why we chose 128 as the size for some of these, while others have 256?

Reply:
I'm not sure.  I have not investigated that far. @CIPop might know.